Oracle/prescribing.sql

```diff
@@ -73,19 +81,24 @@ from dual
 create table prescribing_key as
 select cast(prescribing_seq.nextval as varchar(19)) prescribingid
 , instance_num
-, cast(patient_num as varchar(50)) patient_num
-, cast(encounter_num as varchar(50)) encounter_num
-, provider_id
+, cast(patient_num as varchar(50)) patid
+, cast(encounter_num as varchar(50)) encounterid
+, provider_id rx_providerid
 , start_date
 , end_date
 , concept_cd
 , modifier_cd
-from blueherondata.observation_fact rx
+, case when trim(translate(tval_char, '0123456789.', ' ')) is null then tval_char else null end rx_dose_ordered
```

Comment:
Exclude ranged values.  If I need to comment here, it should probably be a comment in code, right?